libgalois/include/katana/GraphHelpers.h

```diff
@@ -172,7 +172,7 @@ auto
 divideNodesBinarySearch(
```

Comment:
I'm assuming changing this to `DivideNodesBinarySearch` would be way too intrusive
for such a simple commit (to match function convention with the rest of katana)?

Reply:
I would be okay with adding an overload that has the desired name. Then we could remove uses of the old version incrementally before removing the original.

Reply:
Please don't add overloads. It can be confusing. Either remote it or continue using the old one.

Reply:
My issue with removing it is that that would be an interface modification. And an interface modification always requires a tightly coupled enterprise PR. If we instead do an interface addition followed by a migration to the new interface followed by an interface deletion then we can get the same effect without ever having to coordinate PRs across repos.

Reply:
Correction: Either replace with new one or continue using the old one.

I get your point. That's not ideal. If you're doing both additional of new one and removal of old one in quick succession, sure, that's fine.

Reply:
That is my intention. The overload is just a temporary measure to simplify the transition.